src/components/GalleryCard.tsx

```diff
@@ -1,54 +1,50 @@
-
 export type GalleryCardType = {
-    title: string;
-    artist: string;
-    location: string;
-    image: string;
-    altText: string;
-    //once the api is called not optional
-    imageApi?: string;
-    iiifValue?: number;
-    imageId?: string;
-
+  title: string;
+  artist: string;
+  location: string;
+  image: string;
+  altText: string;
+  //once the api is called not optional
+  imageApi?: string;
+  iiifValue?: number;
+  imageId?: string;
 };
 
-function GalleryCard (
-   { title,
-    artist,
-    location,
-    image,
-    altText} : {
-        title: string
-        artist: string
-        location: string
-        image: string
-        altText: string
-    }
-) {
-  
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const createImageURL = (iiifValue: number, imageId: string) => {
-    const url = `https://www.artic.edu/iiif/${iiifValue}/${imageId}/full/843,/0/default.jpg`
-    return url
-
-}
+function GalleryCard({
+  title,
+  artist,
+  location,
+  image,
+  altText,
+}: {
+  title: string;
+  artist: string;
+  location: string;
+  image: string;
+  altText: string;
+}) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const createImageURL = (iiifValue: number, imageId: string) => {
+    const url = `https://www.artic.edu/iiif/${iiifValue}/${imageId}/full/843,/0/default.jpg`;
+    return url;
+  };
 
-    return (
-        
-        <div className="card">
-        <img alt={altText} src={image} className = "art-image"></img>
+  return (
+    <li>
```

Comment:
It's actually better to not include the `<li>` inside the component to make the component more reusable – otherwise it can only be used inside a list.
But that's something you can look into at some point later on